packages/gmp/install

```diff
@@ -3,4 +3,4 @@ set -x
 set -e
 
 apt-get update -qq
-apt-get install -y libgmp10-dev
+apt-get install -y libgmp-dev
```

Comment:
this works on xenial too?

Reply:
Good catches, probably not

Reply:
also.. why are there no tests for these :(

Reply:
lol, because tests came after the initial version of many of these, and they were not circled back to.
test away!